File: #14/lesson_71_FCCamp.py
watchList = [{"Title": "inception", "imdbRating": 8.5}, {"Title": "dark knight", "imdbRating": 9.1}]

def iterate_func(element):
    return element["Title"]

titles = list(map(iterate_func, watchList))
print(titles)


def new_map(function, iterator):
    
    for item in iterator:
        yield function(item)
    
new_titles = list(new_map(iterate_func, watchList))
print(new_titles)

File: #14/lesson_71_FCCamp.js
let watchList = [{"Title": "inception", "imdbRating": 8.5}, {"Title": "Memento", "imdbRating": 8.1}]

let ratings = watchList.map((element, indx, ) => {
    // console.log(element);
    return {
        "title": watchList[indx]["Title"],
        "rating": watchList[indx]["imdbRating"] 
    };
})

console.log(ratings);
